src/fetchers/opensea.js

```diff
@@ -16,16 +16,19 @@ export const fetchCollection = async (chainId, { contract, tokenId }) => {
     try {
       const url =
         chainId === 1
-          ? `https://api.opensea.io/api/v1/asset/${contract}/${tokenId}`
-          : `https://testnets-api.opensea.io/api/v1/asset/${contract}/${tokenId}`;
+          ? `${ process.env.OPENSEA_BASE_URL ?? "https://api.opensea.io" }/api/v1/asset/${contract}/${tokenId}`
+          : `${ process.env.OPENSEA_BASE_TESTNET_URL ?? "https://testnets-api.opensea.io" }/api/v1/asset/${contract}/${tokenId}`;
```

Comment:
I still think you dont need a `OPENSEA_BASE_TESTNET_URL`. you check if you have the `OPENSEA_BASE_URL` and if not you check the chain id to decide on api.opensea / testnests-api.opensea.

`process.env.OPENSEA_BASE_URL ?? chainId === 1 ? https://api.opensea.io : https://testnets-api.opensea.io`